site/soc/software/charts/osh/openstack-compute-kit/nova.yaml

```diff
@@ -80,15 +80,36 @@ data:
     conf:
       ceph:
         enabled: true
-        monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
+        #monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
+        #admin_keyring: {{ ceph_admin_keyring_key }}
         admin_keyring: {{ ceph_admin_keyring_b64key | b64decode }}
         cinder:
-          user: {{ ses_cluster_configuration.cinder.rbd_store_user }}
-          keyring: {{ ses_cluster_configuration.cinder.key }}
+          user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
+          keyring: {{ suse_ses_cinder_pool_key | default( ceph_admin_keyring_b64key | b64decode) }}
```

Comment:
can you explain why we are changing to use suse_ses_cinder_pool_key which seems to be deploy_osh var?